src/app/submission/import-external/submission-import-external.component.spec.ts

```diff
@@ -177,6 +184,326 @@ describe('SubmissionImportExternalComponent test suite', () => {
     });
   });
 
+  describe('handle BE response for search query', () => {
+    const paginatedData: any = {
+      'timeCompleted': 1657009282990,
+      'msToLive': 900000,
+      'lastUpdated': 1657009282990,
+      'state': 'Success',
+      'errorMessage': null,
+      'payload': {
+        'type': {
+          'value': 'paginated-list'
+        },
+        'pageInfo': {
+          'elementsPerPage': 10,
+          'totalElements': 11971608,
+          'totalPages': 1197161,
+          'currentPage': 1
+        },
+        '_links': {
+          'first': {
+            'href': 'https://dspacecris7.4science.cloud/server/api/integration/externalsources/scopus/entries?query=test&page=0&size=10&sort=id,asc'
```

Comment:
Could we avoid using 4Science specific URLs in these tests?  If you want you can use https://demo7.dspace.org.  Or you could just use a fake URL like "https://rest.api/server/" (which some other tests use).

This same comment goes for all the "dspacecris7.4science.cloud" URLs in this PR.

Reply:
I'll just mention that `example.com` is permanently reserved for just such uses.

Reply:
Hi @tdonohue , @mwoodiupui , 
thank you for pointing this out. I have just pushed a fix using example.com as domain. Thank you